src/d/actor/d_a_swc00.cpp

```diff
@@ -3,26 +3,90 @@
 // Translation Unit: d_a_swc00.cpp
 //
 
-#include "d_a_swc00.h"
-#include "dolphin/types.h"
+#include "f_op/f_op_actor_mng.h"
+#include "JSystem/JKernel/JKRHeap.h"
+#include "d/d_procname.h"
+#include "d/d_com_inf_game.h"
 
-/* 00000078-00000180       .text daSwc00_Execute__FP11swc00_class */
-void daSwc00_Execute(swc00_class*) {
-    /* Nonmatching */
+struct swc00_class : public fopAc_ac_c {};
+
+static inline s32 getSwBit(u32 param) {
+    return param & 0xFF;
 }
 
-/* 00000180-00000188       .text daSwc00_IsDelete__FP11swc00_class */
-void daSwc00_IsDelete(swc00_class*) {
-    /* Nonmatching */
+static s32 daSwc00_Execute(swc00_class* i_this) {
```

Comment:
if this is still nonmatching, i think leaving the nonmatching comment would be helpful